Dockerfile

```diff
@@ -1,12 +1,12 @@
 FROM quay.io/hotosm/base-python-image as base
-LABEL version=0.1
-LABEL maintainer="HOT Sysadmin <sysadmin@hotosm.org>"
-LABEL description="Builds backend docker image"
-
-WORKDIR /usr/src/app
+LABEL version=0.2
+LABEL hotosm.org.maintainer="HOT Sysadmin <sysadmin@hotosm.org>"
+LABEL hotosm.org.description="Builds docker image for TM Backend"
```

Comment:
I understand MAINTAINER label is deprecated and we can simply use LABELS for it. But what's the reason for using hotosm.org.maintainer as key? Docker suggests tld.domain.sub.maintainer but I'm not sure if it is standard. If it is, we can switch this to org.hotosm.maintainer.

Documentation about this is sparse though.

Reply:
I don't think there are any standards around label keys as far as I know, we can definitely change it 👍